eventsourcing/src/test/java/org/axonframework/eventsourcing/eventstore/AnyEventTest.java

```diff
@@ -1,32 +0,0 @@
-/*
- * Copyright (c) 2010-2025. Axon Framework
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.axonframework.eventsourcing.eventstore;
-
-import org.junit.jupiter.api.*;
-
-import static org.junit.jupiter.api.Assertions.*;
-
-class AnyEventTest {
```

Comment:
Why have you removed this class?

Reply:
I have no idea how or why this happened :thinking: Obviously going to revert this

Reply:
It's still removed :)